Zend/zend_language_scanner.l

```diff
@@ -2476,8 +2694,13 @@ heredoc_scan_done:
 
 nowdoc_scan_done:
 	yyleng = YYCURSOR - SCNG(yy_text);
+	zend_copy_value(zendlval, yytext, yyleng);
```

Comment:
Travis segfault is due to the use of `zend_copy_value` rather than `ZVAL_STRINGL` here. If yyleng is 1 this is an interned string, which is mprotected if opcache is used.

Reply:
Thanks for the tip. I noticed the segfault, but couldn't figure out how I had messed something up to do with string interning (didn't even think to enable OPcache...).